sdk/keyvault/azcertificates/README.md

```diff
@@ -20,92 +20,17 @@ go get github.com/Azure/azure-sdk-for-go/sdk/azidentity
 ### Prerequisites
 
 * An [Azure subscription][azure_sub]
-* Go 1.18 or later
-* A Key Vault. If you need to create one, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with these commands (replace `"my-resource-group"` and `"my-key-vault"` with your own, unique names):
-
-  (Optional) if you want a new resource group to hold the Key Vault:
-  ```sh
-  az group create --name my-resource-group --location westus2
-  ```
-
-  Create the Key Vault:
-  ```Bash
-  az keyvault create --resource-group my-resource-group --name my-key-vault
-  ```
-
-  Output:
-  ```json
-  {
-      "id": "...",
-      "location": "westus2",
-      "name": "my-key-vault",
-      "properties": {
-          "accessPolicies": [...],
-          "createMode": null,
-          "enablePurgeProtection": null,
-          "enableSoftDelete": null,
-          "enabledForDeployment": false,
-          "enabledForDiskEncryption": null,
-          "enabledForTemplateDeployment": null,
-          "networkAcls": null,
-          "provisioningState": "Succeeded",
-          "sku": { "name": "standard" },
-          "tenantId": "...",
-          "vaultUri": "https://my-key-vault.vault.azure.net/"
-      },
-      "resourceGroup": "my-resource-group",
-      "type": "Microsoft.KeyVault/vaults"
-  }
-  ```
-
-  > The `"vaultUri"` property is the `vaultURL` argument for [NewClient][certificate_client_docs]
-
-### Authenticate the client
-
-This document demonstrates using [DefaultAzureCredential][default_cred_ref] to authenticate as a service principal. However, [NewClient][certificate_client_docs]
-accepts any [azidentity][azure_identity] credential. See the [azidentity][azure_identity] documentation for more information about other credentials.
-
-#### Create a service principal (optional)
-
-This [Azure Cloud Shell][azure_cloud_shell] snippet shows how to create a new service principal. Before using it, replace "my-application" with an appropriate name for your application.
-
-Create a service principal:
-```Bash
-az ad sp create-for-rbac --name http://my-application --skip-assignment
-```
+* A supported Go version (the Azure SDK supports the two most recent Go releases)
```

Comment:
Are these version numbers fixed or can they change in the future? If so, when a new version of Go is released would we drop support for the older one and add support for the newer one right away?

Reply:
I removed the specific version because the requirement will change over time as Google releases new versions (roughly every 6 months). We won't ordinarily ship a module only to upgrade its required Go version.